Assets/MixedRealityToolkit/_Core/Utilities/FastSimplexNoise.cs

```diff
@@ -0,0 +1,501 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities
+{
+    /// <summary>
+    /// A conglomeration of open-source simplex libraries in C# with an emphasis on performance
```

Comment:
Also needs summary tags on all the public functions

Reply:
Were these libraries MIT licensed?

Reply:
Good question, seeing as this already exists in master...